{
  "Severity": "HIGH",
  "UpdatedAt": "2023-02-01T01:36:03Z",
  "Package": {
    "Ecosystem": "MAVEN",
    "Name": "org.apache.iotdb:iotdb-parent"
  },
  "Advisory": {
    "DatabaseId": 201830,
    "Id": "GSA_kwCzR0hTQS1wcDR3LTl4ODItNnI0N84AAxRm",
    "GhsaId": "GHSA-pp4w-9x82-6r47",
    "References": [
      {
        "Url": "https://nvd.nist.gov/vuln/detail/CVE-2023-24830"
      },
      {
        "Url": "https://lists.apache.org/thread/l4fon37687jz5ohgsnz2ko9fv400915t"
      },
      {
        "Url": "https://github.com/advisories/GHSA-pp4w-9x82-6r47"
      }
    ],
    "Identifiers": [
      {
        "Type": "GHSA",
        "Value": "GHSA-pp4w-9x82-6r47"
      },
      {
        "Type": "CVE",
        "Value": "CVE-2023-24830"
      }
    ],
    "Description": "Improper Authentication vulnerability in Apache Software Foundation Apache IoTDB.This issue affects Apache IoTDB: from 0.13.0 before 0.13.3.",
    "Origin": "UNSPECIFIED",
    "PublishedAt": "2023-01-30T18:30:28Z",
    "Severity": "HIGH",
    "Summary": "Apache IoTDB contains Improper Authentication",
    "UpdatedAt": "2023-02-07T04:06:26Z",
    "WithdrawnAt": "",
    "CVSS": {
      "Score": 7.5,
      "VectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:H/A:N"
    }
  },
  "Versions": [
    {
      "FirstPatchedVersion": {
        "Identifier": "0.13.3"
      },
      "VulnerableVersionRange": "\u003e= 0.13.0, \u003c 0.13.3"
    }
  ]
}